### PR TITLE
Fix website links for Huerta Tipográfica

### LINKED ISSUE #5416: Fix website links for Huerta Tipográfica
Hey there, the URL for Huerta Tipográfica have changed and the site now ends with .com instead of .com.ar.

**Describe the issue**
https://huertatipografica.com.ar/ should be replaced with https://huertatipografica.com/

**To Reproduce**
Go to the website for Huerta Tipográfica as listed in some font descriptions and you will see that the link is no longer valid.

**Expected behavior**
The link should take me to the right website.

**Additional context**
I have already made the changes in a seperate fork, let me know if you want me to do a PR.

**Screenshots**
Screen Shot 2022-10-16 at 3 33 41 PM

### PR DESCRIPTION
**Pull request by applecuckoo**
Fixes #5416 

Acme
Alegreya
Alegreya SC
Alegreya Sans
Alegreya Sans SC
Andada Pro
Bitter
Cambo
Gochi Hand
Piazzolla
Telex

### CODE REVIEW

**Comment by RosaWagner** [comment]
Thanks @applecuckoo for the contribution :) Can you list all the modified families in the main comment? (it will make my sandbox reviewing easier)

**Comment by applecuckoo** [comment]
Here they are:

- Acme
- Alegreya
- Alegreya SC
- Alegreya Sans
- Alegreya Sans SC
- Bitter
- Cambo
- Gochi Hand
- Telex

**Comment by applecuckoo** [comment]
Don't know how to fix the FontBakery warnings, also the warning has a typo:
`WARN The last **characther** on DESCRIPTION.en_us.html is not a                
          line-break. Please add it. [code: missing-eof-linebreak]`

**Comment by thlinard** [comment]
> Here they are @RosaWagner:
> 
>     * Acme
> 
>     * Alegreya
> 
>     * Alegreya SC
> 
>     * Alegreya Sans
> 
>     * Alegreya Sans SC
> 
>     * Bitter
> 
>     * Cambo
> 
>     * Gochi Hand
> 
>     * Telex

And:

- Andada Pro
- Piazzolla

And probably also:

- Sahitya

Of which https://fonts.google.com/specimen/Sahitya/about indicates as source "github.com/juandelperal/sahitya" but which is redirected to https://github.com/huertatipografica/sahitya

**Comment by RosaWagner** [comment]
> Don't know how to fix the FontBakery warnings, also the warning has a typo: `WARN The last **characther** on DESCRIPTION.en_us.html is not a line-break. Please add it. [code: missing-eof-linebreak]`

It means you need to add a blank line at the end of the description (this is to prevent something I don't remember)

**Comment by applecuckoo** [comment]
That last warning didn't have "character" spelled correctly.

**Comment by applecuckoo** [comment]
@thlinard Andada Pro and Piazzola don't link to the font foundry website, though I will fix Sahitya to use the new link.

**Comment by applecuckoo** [comment]
there's a lot of errors, I'm combing through it now and will reply with fixes.

**Comment by thlinard** [comment]
> @thlinard Andada Pro and Piazzola don't link to the font foundry website, though I will fix Sahitya to use the new link.

Well, it could be an opportunity to harmonize a little. Instead of:

1. "Piazzolla is designed by Huertatipografica, to contribute see https://github.com/huertatipografica/piazzolla."
2. "To contribute see [github.com/huertatipografica/Andada-Pro](https://github.com/huertatipografica/Andada-Pro)."

We could have, like the others typefaces:

1. "Designed by Juan Pablo del Peral for [Huerta Tipográfica](https://www.huertatipografica.com/)."
2. "Designed by Carolina Giovagnoli for [Huerta Tipográfica](https://www.huertatipografica.com/)."

**Comment by RosaWagner** [comment]
@applecuckoo since you are only changing descriptions, it's okay to ignore fails related to fonts. I am happy with current changes.

File: ofl/sahitya/METADATA.pb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: "Sahitya"
-designer: "Juan Pablo del Peral"
+designer: "Juan Pablo del Peral, Huerta Tipográfica"
```
Comment (RosaWagner) [inline ofl/sahitya/METADATA.pb:2]: @applecuckoo please remove changes to METADATA.pb in this PR. If the author gives consent to change the credit then it should be made at their demand, and in a separated PR :)

Comment (applecuckoo) [reply ofl/sahitya/METADATA.pb]: Reverted.

**Pull request merged**